commercial/app/model/feeds/FeedReader.scala

```diff
@@ -119,13 +115,12 @@ class FeedReader(wsClient: WSClient) extends Logging {
   }
 }
 
-
 case class FeedRequest(feedName: String,
                        switch: conf.switches.Switch,
                        url: String,
                        parameters: Map[String, String] = Map.empty,
                        timeout: Duration = 2.seconds,
-                       responseEncoding: Option[String] = None)
+                       responseEncoding: String)
```

Comment:
`BookFinder` is the only module that uses the `FeedRequest` and `FeedReader` classes - it can probably be cleaned up at some point but it is beyond the scope of this PR.